broker_app.py

```diff
@@ -32,7 +32,7 @@
 def add_routes(app):
     @app.route('/', methods=['GET'])
     def index():
-        new_ui_url = os.environ.get('INGEST_UI')
+        new_ui_url = os.getenv('INGEST_UI')
```

Comment:
I don't think defaulting to an empty value is correct. It is good for testing, maybe, but can mean that the system will be in an inconsistent state. I'd rather to fail fast on these occasions, and have the tests prepare the required environment fixture.

Reply:
but also it means that our code is less useful to consumers who might not have set those environment variables that we are not making explicit in the constructors of our classes